package.json

```diff
@@ -58,6 +58,7 @@
         "postgres": "^3.4.4",
         "tailwind-merge": "^2.5.2",
         "tailwind-variants": "^0.2.1",
+        "vaul-svelte": "^0.3.2",
```

Comment:
What's the advantage of this drawer over the shadcn svelte one? 
https://www.shadcn-svelte.com/docs/components/drawer

Reply:
Shadcn depends upon that library for the drawer.

> Drawer is built on top of [Vaul Svelte](https://vaul-svelte.com/), which is a Svelte port of [Vaul](https://vaul.emilkowal.ski/) by [Emil Kowalski](https://twitter.com/emilkowalski_).